src/Illuminate/Auth/Passwords/PasswordBroker.php

```diff
@@ -35,15 +45,15 @@ class PasswordBroker implements PasswordBrokerContract
     /**
      * Create a new password broker instance.
      *
-     * @param  \Illuminate\Auth\Passwords\TokenRepositoryInterface  $tokens
+     * @param  \Illuminate\Contracts\Foundation\Application  $app
```

Comment:
That makes sense. Changed in https://github.com/laravel/framework/pull/17499/commits/c709b25f6b10e0a247cc46cdbfaf0dc1b7091d52.